recipes/cythonbiogeme/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "cythonbiogeme" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cythonbiogeme-{{ version }}.tar.gz
+  sha256: f139342944c849b47ce5aaa548e68aa179c195de93ab76be78353f4309684d0e
```

```suggestion
  sha256: f139342944c849b47ce5aaa548e68aa179c195de93ab76be78353f4309684d0e
  patches:
    - setup.py.patch
```

and include a patch file with these contents:

```
--- setup.py
+++ setup.py
@@ -435,12 +435,4 @@
     raise ValueError('invalid boolean string %r' % value)
 
 
-if platform.system() == "Darwin":
-    os.environ["CC"] = "clang++"
-    os.environ["CXX"] = "clang++"
-else:
-    os.environ["CC"] = "g++"
-    os.environ["CXX"] = "g++"
-
-
 setup()
```

---

Setting those in cythonbiogeme's setup.py file prevents the use of environment variables to control the compilers used.